crate_universe/src/utils/starlark/serialize.rs

```diff
@@ -23,3 +102,25 @@ impl Serialize for ExportsFiles {
         call.end()
     }
 }
+
+impl Data {
+    pub fn is_empty(&self) -> bool {
+        self.glob.is_empty() && self.select.is_empty()
+    }
+}
+
+impl Serialize for Data {
+    fn serialize<S>(&self, serializer: S) -> Result<S::Ok, S::Error>
+    where
+        S: Serializer,
+    {
+        let mut plus = serializer.serialize_tuple_struct("+", MULTILINE)?;
```

Comment:
It would be nice to only do the plus-ing if both are non-empty (again, we can do in a follow-up)

Reply:
I think that's equivalent to what this is already doing. This as currently written serializes to:

| `self.glob` | `self.select.common` | `self.select.selects` | serialized |
|---|---|---|---|
| empty | empty | empty | `[]` |
| empty | empty | non-empty | `select(…)` |
| empty | non-empty | empty | `[…]` |
| empty | non-empty | non-empty | `[…] + select(…)` |
| non-empty | empty | empty | `glob(…)` |
| non-empty | empty | non-empty | `glob(…) + select(…)` |
| non-empty | non-empty | empty | `glob(…) + […]` |
| non-empty | non-empty | non-empty | `glob(…) + […] + select(…)` |